### PR TITLE
Check in `fuzzable.wasm` to git

### PR DESCRIPTION
**Pull request by rbehjati**
This is another attempt at fixing the ClusterFuzz failures (https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=34701).

This change list:
* Checks in `fuzzable.wasm` to git
* Uses `include_bytes!` to include the Wasm module in the binay
* Updates the `run-cargo-fuzz` command to also build the required wasm files. This ensures that `fuzzable.wasm` will remain up-to-date.

### CODE REVIEW

File: runner/src/main.rs [hunk 1]
```diff
@@ -224,7 +318,13 @@ pub fn run_fuzz_targets_in_crate(path: &PathBuf, opt: &RunFuzzTargets) -> Step {
                 name: binary.name.clone(),
                 command: Cmd::new_in_dir(
                     "cargo-fuzz",
-                    spread!["run".to_string(), binary.name.clone(), "--target=x86_64-unknown-linux-gnu".to_string(), "-O".to_string(), "--".to_string(), ...opt.args],
+                    spread!["run".to_string(),
+                        binary.name.clone(),
+                        "--target=x86_64-unknown-linux-gnu".to_string(),
+                        "-O".to_string(),
```
Comment (ipetr0v) [inline runner/src/main.rs:324]:
```suggestion
                        "--release".to_string(),
```

File: runner/src/main.rs [hunk 2]
```diff
@@ -177,7 +177,101 @@ fn run_tests_tsan() -> Step {
...
+            format!("{}/{}.wasm", &config.out_dir, &config.name),
+        ],
+    );
+
+    Step::Multiple {
+        name: "copy to dir".to_string(),
+        steps: vec![
+            Step::Single {
+                name: "mkdir".to_string(),
+                command: Cmd::new("mkdir", vec!["-p", &config.out_dir]),
```
Comment (ipetr0v) [inline runner/src/main.rs:264]:
```suggestion
                command: Cmd::new("mkdir", vec!["--parents", &config.out_dir]),
```

**Review by ipetr0v — APPROVED** [review]

File: runner/src/main.rs (hunk 2 above)
Comment (rbehjati) [reply runner/src/main.rs]: Done.

File: runner/src/main.rs (hunk 1 above)
Comment (rbehjati) [reply runner/src/main.rs]: Done.

**Review by rbehjati — COMMENTED** [review]
Thanks!

**Pull request merged**